frontend/public/components/utils/headings.tsx

```diff
@@ -30,13 +30,14 @@ export const BreadCrumbs: React.SFC<BreadCrumbsProps> = ({breadcrumbs}) => (
     }) }
   </ol>);
 
-const ActionButtons: React.SFC<ActionButtonsProps> = ({actionButtons}) => <div className="co-action-buttons">
+const ActionButtons: React.SFC<ActionButtonsProps> = ({actionButtons}) => <React.Fragment>
   {_.map(actionButtons, (actionButton, i) => {
     if (!_.isEmpty(actionButton)) {
-      return <button className={`btn ${actionButton.btnClass} co-action-buttons__btn`} onClick={actionButton.callback} key={i}>{actionButton.label}</button>;
+      return <Button variant="primary" onClick={actionButton.callback} key={i}>{actionButton.label}</Button>;
```

Comment:
todo: we can look at PF4 Button variations for this when mapping `actionButton.btnClass`
http://patternfly-react.surge.sh/patternfly-4/components/button/